src/main/java/uk/gov/hmcts/reform/blobrouter/config/FlywayConfig.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.blobrouter.config;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.flyway.FlywayMigrationStrategy;
+import org.springframework.context.annotation.Bean;
+
+import java.util.stream.Stream;
+
+public class FlywayConfig {
```

Comment:
Might as well be super safe by adding following:

```java
@AutoConfigureAfter({
    DataSourceAutoConfiguration.class
})
@AutoConfigureBefore({
    FlywayAutoConfiguration.class
})
@Configuration
@ConditionalOnClass(Flyway.class)
@ConditionalOnProperty(prefix = "spring.flyway", name = "enabled", matchIfMissing = true)
```

Reply:
done